Makefile

```diff
@@ -233,6 +233,12 @@ check:
 	@echo "goimports"
 	@! goimports -l . | grep -vF 'No Exceptions'
 
+.PHONY: unused
+unused:
+# The -fields option generates false-positives for embedded mutex and
+# related types. See https://github.com/dominikh/go-unused/issues/18.
```

Comment:
Thanks for the note. Looks like the Mutex false-positives are all gone. I did find a new false-positive: https://github.com/cockroachdb/cockroach/blob/master/storage/engine/rocksdb/rocksdb.go#L38. That method (`rocksDBLog`) is exported to C so it appears unused by Go code.